docs/HowToSetEnv.md

```diff
@@ -6,44 +6,81 @@ Get the build in binaries and libraries, [gcc-arm-none-eabi-6-2017-q1-update-*OS
 Untar the gcc-arm-none-eabi-6-2017-q1-update-*OS Type*.tar.bz2 and export the path like
 
 ```bash
-tar xvjf gcc-arm-none-eabi-6-2017-q1-update-[OS Type].tar.bz2
-export PATH=<Your Toolchain PATH>:$PATH
+	/>  tar xvjf gcc-arm-none-eabi-6-2017-q1-update-[OS Type].tar.bz2
```

Comment:
I prefer older formating because I can paste it to terminal
IMHO,  using ">" in prompt is dangerous because It can create (and overide) files , when pasted by mistake

Reply:
You mean like the /$ sign?

Reply:
No just like it was:

```bash
tar ...
```

$ can be good to differentiate user vs root '#' but there is no point here

prefixing with "/" is your own preference of prompt,
there is no reason to spread it in TizenRT docs

Reply:
I agree @rzr 's opinion. But if it is difficult to know what it is, how about "$" like Linux prompt?
Because I already know and understand TizenRT, I don't feel difficulty to understand it. So you guys opinion is very important to me.

Reply:
I agree with @sunghan-chang . If you have instructions with shell commands and config files it's a good idea to make clear which is which. Perhaps '$' should be the way forward?